src/main/java/mate/controller/IndexController.java

```diff
@@ -2,10 +2,12 @@
 
 import java.io.IOException;
 import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+@WebServlet(urlPatterns = {"/", "/index", "/create"})
```

Comment:
I think this is redundant 
```suggestion
@WebServlet(urlPatterns = {"/", "/index"})
```